src/components/modeler/Modeler.vue

```diff
@@ -1,5 +1,19 @@
 <template>
   <b-card-body class="overflow-hidden position-relative p-0 vh-100" data-test="body-container">
+    <b-row class="w-100 m-0">
```

Comment:
Could we make the row wrapper part of the tool-bar component? What do you think?